### PR TITLE
AMB-332 Implement Sync wrap on Update endpoints

### PR DESCRIPTION
**Pull request by matthewaclark21**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: api_tests/tests/conftest.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def setup(request):
     # Return patient to previous state
 
     if hasattr(request.cls, 'token'):
-
+        # pass
```
Comment (artronics) [inline api_tests/tests/conftest.py:67]: We can delete this line.

File: api_tests/tests/reasonable_adjustment_flag/test_happy_cases.py [hunk 2]
```diff
@@ -307,9 +313,12 @@ def test_list_post(self):
         assert_that(expected_status_code).is_equal_to(response.status_code)
 
     @pytest.mark.happy_path
+    @pytest.mark.debug
```
Comment (artronics) [inline api_tests/tests/reasonable_adjustment_flag/test_happy_cases.py:316]: We can delete this line.

File: azure/azure-pr-pipeline.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ resources:
     - repository: common
       type: github
       name: NHSDigital/api-management-utils
-      ref: refs/heads/edge
+      ref: refs/heads/master
```
Comment (artronics) [inline azure/azure-pr-pipeline.yml:11]: This should be `edge`. after Platforms merged `master` to `edge`

File: manifest_template.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      products:
+{% set approval_type = env.approval_type | default('auto') %}
+{% for mode in ACCESS_MODES %}
+      - name: reasonable-adjustment-flag-{{ env.name }}{{ mode.nameSuffix }}
+        approvalType: {{ approval_type }}
+        attributes:
+          - name: access
+            value: public
+          - name: ratelimit
+            value: {{ env.ratelimit | default('50ps') }}
```
Comment (artronics) [inline manifest_template.yml:55]: Default value should be `5ps`

File: proxies/live/apiproxy/policies/SpikeArrest.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <SpikeArrest name="SpikeArrest">
   <DisplayName>SpikeArrest</DisplayName>
-  <Rate ref="apiproduct.ratelimit">5ps</Rate>
+  <Rate ref="apiproduct.ratelimit">50ps</Rate>
```
Comment (artronics) [inline proxies/live/apiproxy/policies/SpikeArrest.xml:4]: This should be `5ps` by default.

File: proxies/live/apiproxy/targets/poll.xml [hunk 6]
```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (artronics) [inline proxies/live/apiproxy/targets/poll.xml:1]: I'm not sure what we decided to do with this file. Keep it as a duplication of `ig3.xml` or add logic to change `<Path>` dynamically.

**Comment by gjugpal** [comment]
Can we add the integration marker to this test too please `test_duplicate_consent_record` -- it's in the error tests file

**Review by RyanThomas1214 — APPROVED** [review]

**Pull request merged**